ui/partials/logviewer/lvLogLines.html

```diff
@@ -2,9 +2,10 @@
      ng-if="loading"> Loading... </div>
 
 <div ng-repeat="lv_line in displayedLogLines"
-     ng-class="{'text-danger': (lv_line.hasError == true),
-                'lv-selected-lines': (displayedStep.order === lv_line.index)}"
      class="lv-log-line"
      line="{{ ::lv_line.index }}">
-  <span class="lv-line-text">{{ ::lv_line.text }}</span>
+     <div class="lv-line-num" ng-click="click(lv_line, $event)"> {{::lv_line.index}}  </div>
+     <div ng-class="{'text-danger': (lv_line.hasError == true),
+                     'lv-selected-lines': (lv_line.index >= selectedBegin && lv_line.index <= selectedEnd)}"
+            class="lv-line-text"> {{::lv_line.text}}</div>
```

Comment:
Minor nit, if we can align `class=` here underneath the `ng-class` for readability, that would be great.

eg.
```
  ng-class=

  class=
```